resources/github_pr.go

```diff
@@ -0,0 +1,286 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+var (
+	tokenFile   = flag.String("token_file", "", "File containing Auth Token.")
+	owner       = flag.String("owner", "istio", "Github Owner or org.")
+	repo        = flag.String("repo", "", "Github repo within the org.")
+	base        = flag.String("base", "stable", "The base branch used for PR.")
+	head        = flag.String("head", "master", "The head branch used for PR.")
+	fastForward = flag.Bool("fast_forward", false, "Creates a PR updating Base to Head.")
+	verify      = flag.Bool("verify", false, "Verifies PR on Base and push them if success.")
+	GH          = newGhConst()
```

Comment:
That s what I did at the beginning and I needed to string pointer which made the rest of the code ugly.